packages/template-retail-react-app/app/utils/utils.js

```diff
@@ -292,3 +292,34 @@ export const resolveLocaleFromUrl = (url) => {
         (locale) => locale.alias === defaultLocale || locale.id === defaultLocale
     )
 }
+
+const toCamel = (str) => {
+    if (str.startsWith('_') || str.startsWith('c_')) {
+        return str
+    }
+    return str.replace(/([-_][a-z])/gi, ($1) => {
+        return $1.toUpperCase().replace('-', '').replace('_', '')
+    })
+}
+
+const isObject = (obj) => {
+    return obj === Object(obj) && !Array.isArray(obj) && typeof obj !== 'function'
+}
+
+export const keysToCamel = (obj) => {
```

Comment:
```suggestion
export const camelizeObject = (obj) => {
```
(The verb "camelize" makes me giggle, but probably isn't the best choice for a name.)